src/mount_efs/__init__.py

```
@@ -844,6 +911,80 @@ def get_aws_security_credentials_from_webidentity(
+
+    try:
+        with open(token_file, "r") as f:
+            token = f.read().strip()
+            if "\r" in token or "\n" in token:
+                if is_fatal:
+                    unsuccessful_resp = (
+                        "AWS Container Auth Token contains invalid characters"
+                    )
+                    fatal_error(unsuccessful_resp, unsuccessful_resp)
+                return None, None
+    except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)